app/models/user.rb

```diff
@@ -7,4 +7,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   include DeviseTokenAuth::Concerns::User
+  has_many :rents, dependent: :destroy
```

Comment:
A Book has many rents? If Book is a single physical entity, it can be rent it for many users at same time? What say Apiary for book show controller, Rent param is an array? Book has an atributte for quantity? What do you say about it? ¿is fine or not? ¿Why?